testing/time.ts

```diff
@@ -211,7 +211,7 @@ let dueTree: RedBlackTree<DueNode>;
  * });
  * ```
  */
-export class FakeTime {
+export class FakeTime implements Disposable {
```

Comment:
This class was implementing Disposable already but didn't explicitly say that it was. I updated it to show that it implements Disposable to make it more clear that it is usable.

Reply:
Good catch! Though, let's do this in a different PR. We could probably do PR that ensures that `[Async]Disposable` is implemented across the codebase where necessary.